src/instructions/canis_lupus_meets_felis_silvestris.md

```diff
@@ -12,7 +12,7 @@
 
 Welcome to the *Canis lupus* meets *Felis silvestris* use case.
 
-In this example the GBIF occurrence data of *Canis lupus* and *Felis silvestris* are cut to the extent of Germany and linked to the land use classification of the Ecosystematlas.
+In this example the GBIF occurrence data of *Canis lupus* and *Felis silvestris* are cut to the extent of Germany and linked to the land use classification of the IÖR land use classification.
```

Comment:
Instead of cut you could also say filter, again since it is the name of the operator and is unnecessary to introduce additional terms